kyuubi-util-scala/src/main/scala/org/apache/kyuubi/util/reflect/ReflectUtils.scala

```diff
@@ -93,11 +93,13 @@ object ReflectUtils {
     } catch {
       case e: Exception =>
         val candidates =
-          (clz.getDeclaredMethods ++ clz.getMethods).map(_.getName).distinct.sorted
-        val argClassesNames = argClasses.map(_.getClass.getName)
+          (clz.getDeclaredMethods ++ clz.getMethods)
+            .map(m => s"${m.getName}(${m.getParameterTypes.map(_.getName).mkString(", ")})")
```

Comment:
```suggestion
            .map(m => s"${m.getName}(${m.getParameterTypes.map(_.getSimpleName).mkString(", ")})")
```

A simple class name should be good enough for debugging. Full class name for common classes (other than primitive types) is too long for multiple params of all methods.